includes/functions.php

```diff
@@ -2426,7 +2421,154 @@ function tsml_date_localised($format, $timestamp = null)
     $datetime->setTimezone(new DateTimeZone($timezone));
     return $datetime->format($format);
 }
-/* ******************** end of data_source_change_detection ******************** */
+
```

Comment:
In effect this new method 
- tries to map existing meetings to the incoming ones
- for matches, check for changes
- return 3 arrays, documented below